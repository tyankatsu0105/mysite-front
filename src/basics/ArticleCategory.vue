<template>
	<g-link
		:to="to"
		class="ArticleCategory"
	>
		{{ text }}
	</g-link>
</template>

<script>
export default {
  props: {
    text: {
      type: String,
      requirer: true
    },
    to: {
      type: String,
      requirer: true
    }
  }
};
</script>

<style scoped lang='scss'>
.ArticleCategory {
  display: inline-block;
}
</style>
