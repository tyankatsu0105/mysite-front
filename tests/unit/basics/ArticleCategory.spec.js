import { shallowMount } from "@vue/test-utils";
import ArticleCategory from "@/basics/ArticleCategory.vue";

const propsData = {
  text: "aaaaa",
  to: "hogehoge.com"
};

describe("ArticleCategory", () => {
  it("render", () => {
    const wrapper = shallowMount(ArticleCategory, {
      propsData
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
