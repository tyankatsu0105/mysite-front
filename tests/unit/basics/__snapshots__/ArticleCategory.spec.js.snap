// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleCategory render 1`] = `
<g-link to="hogehoge.com" class="ArticleCategory">
  aaaaa
</g-link>
`;
